src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements Drawable, HasArea {
+    private final double bottomBase;
+    private final double topBase;
+    private final double height;
+
+    IsoscelesTrapezoid(String color, double bottomBase, double topBase, double height) {

```

Comment:
same for other figures
```suggestion
    public IsoscelesTrapezoid(String color, double bottomBase, double topBase, double height) {
```